### DISCUSSION TITLE
Setting up a SQL chain

### DISCUSSION THREAD

**Discussion by danieljqs1** [FlowiseAI/Flowise#441] [Q&A]
Hi!

I'm struggling with the set up of the SQL Chain. I would really love some guidance. 

I uploaded a valid sqlite database (checked it with an online data viewer), but I get very weird responses.

Thanks a lot!!

Attached some images:
![image](https://github.com/FlowiseAI/Flowise/assets/136047714/4397d8c4-ab5f-426a-9244-bfd6806ed373)
![image](https://github.com/FlowiseAI/Flowise/assets/136047714/89b2cb3b-eb87-4427-82cc-c7302e1ab443)

**Comment by HenryHengZJ** [comment]
you can't set an online path, it has to be a file path to your database like `C:/Users/chinook.db`.

see more info [here](https://js.langchain.com/docs/modules/chains/other_chains/sql)
